src/main/java/ch/njol/skript/conditions/CondIsBlock.java

```diff
@@ -0,0 +1,50 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.conditions;
+
+import org.bukkit.inventory.ItemStack;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+
+@Name("Is Block")
+@Description("Checks whether an item is a block")
+@Examples({"player's held item is a block", "iron pickaxe isn't a block"})
+@Since("INSERT VERSION")
+public class CondIsBlock extends PropertyCondition<ItemStack> {
+	
+	static {
+		register(CondIsBlock.class, "[a] block", "itemstacks");
```

Comment:
What do you think about this?
```suggestion
		register(CondIsBlock.class, "([a] block|blocks)", "itemstacks");
```
`{_items::*} are blocks`

Reply:
Should be applied to the other conditions too

Reply:
For most of the others just one form should work, like `player is alive`, `all players are alive` (if such condition even exists.. :eyes:). But yeah, there might be some places where this could also be added

Reply:
That makes a lot of sense, Ill make some changes to it right now (including examples)